src/client/eppo-client.ts

```diff
@@ -307,7 +314,7 @@ export default class EppoClient {
     );
 
     const pollingCallback = async () => {
-      if (await this.flagConfigurationStore.isExpired()) {
```

Comment:
Is one of the goals here to never access the config store directly to inquire about its state?

Reply:
That's right. It moves all the hydration and metadata and entry management into one place where the `EppoClient` doesn't need to be concerned with the inner workings.